Sign_Up_Demo/Controllers/SignUpViewController.swift

```diff
@@ -0,0 +1,54 @@
+import UIKit
+class SignUpViewController: UIViewController {
+    // MARK: Outlets
+    @IBOutlet weak var SignUpView: SignUpView!
+    // MARK: Lifecycle Methods
```

Comment:
Put a 1 line break to make the code more readable.  Put 1 line break above `//MARK`.